src/components/nav.php

```diff
@@ -199,8 +199,10 @@
         $categories = fetch('SELECT * FROM product_categories LIMIT 10');
         if ($categories) {
           foreach ($categories as $category) {
+            
             echo '
-            <a href="/catalog/products?category=' . $category['name'] . '" class="group flex flex-col gap-2 items-center">
+            
+            <a href="/catalog/products?category=' . $category['id'] . '" class="group flex flex-col gap-2 items-center">
```

Comment:
Use the category name instead of the id

Reply:
does not work when i use category name because it doesn't show any product then.